contracts/BC_fusion/BSCGovernor.sol

```diff
@@ -0,0 +1,132 @@
+    ) public view override(GovernorUpgradeable, IGovernorUpgradeable, GovernorTimelockControlUpgradeable) returns (ProposalState) {
+        return super.state(proposalId);
+    }
+
+    function propose(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        string memory description
+    ) public override(GovernorUpgradeable, GovernorCompatibilityBravoUpgradeable, IGovernorUpgradeable) returns (uint256) {
+        return super.propose(targets, values, calldatas, description);
+    }
+
+    function cancel(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        bytes32 descriptionHash
+    ) public override(GovernorUpgradeable, GovernorCompatibilityBravoUpgradeable, IGovernorUpgradeable) returns (uint256) {
+        return super.cancel(targets, values, calldatas, descriptionHash);
+    }
+
+    function _execute(
+        uint256 proposalId,
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        bytes32 descriptionHash
+    ) internal override(GovernorUpgradeable, GovernorTimelockControlUpgradeable) {
+        super._execute(proposalId, targets, values, calldatas, descriptionHash);
+    }
+
+    function _cancel(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        bytes32 descriptionHash
+    ) internal override(GovernorUpgradeable, GovernorTimelockControlUpgradeable) returns (uint256) {
+        return super._cancel(targets, values, calldatas, descriptionHash);
+    }
+
+    function _castVote(
+        uint256 proposalId,
+        address account,
+        uint8 support,
+        string memory reason,
+        bytes memory params
+    )
+    internal
+    override(GovernorUpgradeable, GovernorPreventLateQuorumUpgradeable)
+    returns (uint256)
+    {
+        return GovernorPreventLateQuorumUpgradeable._castVote(
+            proposalId, account, support, reason, params
+        );
+    }
+
+    function _executor() internal view override(GovernorUpgradeable, GovernorTimelockControlUpgradeable) returns (address) {
+        return super._executor();
+    }
+
+    function proposalThreshold()
+    public
+    view
+    override(GovernorSettingsUpgradeable, GovernorUpgradeable)
+    returns (uint256)
+    {
+        return GovernorSettingsUpgradeable.proposalThreshold();
+    }
+
+    function proposalDeadline(uint256 proposalId)
+    public
+    view
+    override(IGovernorUpgradeable, GovernorUpgradeable, GovernorPreventLateQuorumUpgradeable)
+    returns (uint256)
+    {
+        return GovernorPreventLateQuorumUpgradeable.proposalDeadline(proposalId);
+    }
+
+    function supportsInterface(
+        bytes4 interfaceId
+    ) public view override(GovernorUpgradeable, IERC165Upgradeable, GovernorTimelockControlUpgradeable) returns (bool) {
+        return super.supportsInterface(interfaceId);
+    }
+}
```

Comment:
I would like to change some parameters of this contracts through governance.

Reply:
added